src/components/main/MainContent.jsx

```diff
@@ -1,68 +1,134 @@
+import { useState, useEffect } from 'react'
+import { useNote, useAuth } from 'context/index'
+import { Note } from './Note'
+import axios from 'axios'
+import { colors } from './colors'
+
 function MainContent() {
+  const { note, noteDispatch, notes, setNotes } = useNote()
+  const [showPalette, setShowPalette] = useState(false)
+  const [createNote, setCreateNote] = useState(false)
+
+  const { encodedToken } = useAuth()
+
+  const saveNoteHandler = async e => {
```

Comment:
You can add this function to some other file to keep it clean.

Reply:
sure will do it